private/model/api/api.go

```diff
@@ -803,6 +819,7 @@ func (a *API) InterfaceGoCode() string {
 		a.AddSDKImport("aws")
 	}
 	a.AddImport(a.ImportPath())
+	a.AddSDKImport("service", a.PackageName(), ServiceTypesPkgName)
```

Comment:
This should use `a.AddImport(a.ImportPath(), ServiceTypesPkgName)`

Or even better would be to create a utility that does this directly

.e.g 
```go
a.AddSDKServiceTypesImport()
a.AddSDKServiceEnumsImport()
```